### ISSUE TITLE
ability to change ip 

### ISSUE THREAD

**Issue by wulfric93** [yonggekkk/warp-yg#65]
hi there hope you doing well 
I've installed this on my VPS (Iranian VPS)
when i enable SOCKS5 proxy of warp , its Ip is from Cloudflare in Iran 
is it possible to change the given Ip to other country like Germany or any other country except Iranian Cloudflare Ip ?

**Comment by yonggekkk** [comment]
Nope! Depending on the ip region of the vps, warp assigns ip addresses close by! This is determined by the mechanism by which warp assigns ip's.

**Comment by wulfric93** [comment]
there is no way ?

**Comment by wulfric93** [comment]
here is the ChatGPT anwser to me :
![image](https://github.com/yonggekkk/warp-yg/assets/117269334/7df7e510-bc56-4e90-8436-d7e2e10ca940)

**Comment by karim20012** [comment]
> here is the ChatGPT answer to me : ![image](https://private-user-images.githubusercontent.com/117269334/261821506-7df7e510-bc56-4e90-8436-d7e2e10ca940.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE2ODkzMDksIm5iZiI6MTcwMTY4OTAwOSwicGF0aCI6Ii8xMTcyNjkzMzQvMjYxODIxNTA2LTdkZjdlNTEwLWJjNTYtNGU5MC04NDM2LWQ3ZTJlMTBjYTk0MC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIwNFQxMTIzMjlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mYmFiYjI1OTZkYzM3OTU4MzNmMzg4MWU2YTEwNWIxOGYzODIwMzFiYzc1OTg5YjcwNmUwNjI3ZTgxZDNmMzk0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.sEWKnn-PCpjeWhfU7QyUVkQjJ3AayVGu4UZB6DLfLFI)

Did this method work?
